ckanext/dgu/schema.py

```diff
@@ -141,6 +141,8 @@
 
 update_frequency_options = ['never', 'discontinued', 'annual', 'quarterly', 'monthly']
 
+publication_delay_options = ['minimal', 'good', 'reasonable', 'extreme']
```

Comment:
Since these words are not meaningful, it would be better to store tuples of the word and its meaning. Then you can display the meaningful version when viewing the dataset.

Also, I think ODI have chosen judgmental words here and we can't really do the same. How about 'short', 'medium' and 'long'. Or you could be somewhat meaningful: 'under half the period', 'period' and 'more than the period' (although you still would want to display the most meaningful version I think).